types/reactstrap/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for reactstrap 5.0
+// Type definitions for reactstrap 6.0
```

Comment:
If you change the major number for a package, you should ensure there is a version-specific package (in this case `v5`) with a copy of the declarations for the previous version. See the guidance at https://github.com/DefinitelyTyped/DefinitelyTyped#i-want-to-update-a-package-to-a-new-major-version, as well as https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/reactstrap/v4 for an example.

Reply:
Personally, I don't think we should maintain a separate v5 folder. There's no breaking changes from v5 to v6, and v6 is a fairly minor update.

v4 is unsupported on the bootstrap side of things, and isn't maintained anymore as well.

Reply:
Hey @rbuckton, very sorry to ping you again, what would be your preferred way, regarding the comments?

This issue is a massive blocker for me, I would love to get it merged :)

Reply:
I think it would be best if the types also used 6 as the major version, it reduces confusion over which `@types` version pairs with the normal version. However, there should still be a `v5` folder if the major is bumped.